### PR TITLE
[TEST] DO NOT PROCESS

### PR DESCRIPTION
**Pull request by slavvka**
### Fixed Issues (if relevant)

https://github.com/magento/magento2/issues/23314

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by sdzhepa — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, thank you for the review.
[ENGCOM-5374](https://jira.corp.magento.com/browse/ENGCOM-5374) has been created to process this Pull Request
:eight_spoked_asterisk: @sdzhepa, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by engcom-tester — COMMENTED** [review]
test comment

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a
 
 *	[Installation Guide](https://devdocs.magento.com/guides/v2.3/install-gde/bk-install-guide.html).
 
-<h2>Contributing to the Magento 2 Code Base</h2>
+<h2>Contributinrfg to the Magento 2 Code Base</h2>
```
Comment (engcom-tester) [inline README.md:14]: line comment

**Comment by engcom-tester** [comment]
test comment

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a
 
 *	[Installation Guide](https://devdocs.magento.com/guides/v2.3/install-gde/bk-install-guide.html).
```
Comment (engcom-tester) [inline README.md:12]: test commet

**Comment by engcom-tester** [comment]
@magento I am working on it

**Pull request closed without merging**